src/main/java/datastax/astra/migrate/CopyJobSession.java

```diff
@@ -126,13 +123,24 @@ public void getDataAndInsert(BigInteger min, BigInteger max) {
         }
     }
 
+    public synchronized void printCounts(boolean isFinal) {
+        String msg = "TreadID: " + Thread.currentThread().getId();
```

Comment:
#nitpick #trivial I would change `TreadID` to `ThreadID` on this typo?

Reply:
@msmygit Great inputs. Can you please review one more time. Thx!